test/test_transforms.py

```diff
@@ -1314,6 +1314,25 @@ def test_gaussian_blur_asserts():
         transforms.GaussianBlur(3, "sigma_string")
 
 
+def test_gaussian_noise():
+    np_img = np.ones((100, 100, 3), dtype=np.uint8) * 255
+    img = F.to_pil_image(np_img, "RGB")
+    out = transforms.GaussianNoise(2.0, (0.1, 2.0))(img)
+    assert F._is_pil_image(out) is True
```

Comment:
I don't think so we need to assert this as this is independent job of `F._is_pil_image` which should just work.

Reply:
@parth-shastri to check the output type, please do `isinstance(out, PIL.Image.Image)`

Reply:
@parth-shastri please address this comment